draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -841,6 +867,26 @@ the SD-JWT.
 
 # Privacy Considerations {#privacy_considerations}
 
+## Confidentiality during Transport
+
+If the SD-JWT and associated Disclosures are transmitted over an insecure
+channel during issuance or presentation, an adversary may be able to
+intercept and read the End-User's personal data or correlate the information with previous uses of the same SD-JWT.
+
+Usually, transport protocols for issuance and presentation of credentials
+are designed to protect the confidentiality of the transmitted data, for
+example, by requiring the use of TLS.
+
+This specification therefore considers the confidentiality of the data to be
+provided by the transport protocol and does not specify any encryption
+mechanism.
+
+Implementers MUST ensure that the transport protocol provides confidentiality,
+if the privacy of End-User data or correlation attacks are a concern. Implementers MAY define an
+envelope format (such as described in (#enveloping) or nesting the SD-JWT Combined Format as
+the plaintext payload of a JWE) to encrypt the SD-JWT
+and associated Disclosures when transmitted over an insecure channel.
```

Comment:
```suggestion
If the privacy of End-User data or correlation attacks are a concern, 
Implementers MUST ensure that the transport protocol provides confidentiality. 
When transmitted over an insecure channel, implementers MAY also define an
envelope format (such as described in (#enveloping) or nesting the SD-JWT Combined Format as
the plaintext payload of a JWE) to encrypt the SD-JWT
and associated Disclosures.
```

Reply:
This changes the meaning (in an unintended way). Implementers may define an envelope format even for secure channels. 

(Also, the second sentence may imply that Implementers are sometimes transmitted over insecure channels... )